src/Xrefcheck/Util.hs

```diff
@@ -24,12 +24,12 @@ instance Pretty Builder where
     style s = build @Text . style s . fmt
 
 nameF' :: Builder -> Builder -> Builder
-nameF' a b = nameF (style Faint a) b
+nameF' a = nameF (style Faint a)
```

Comment:
Not a huge deal in this particular instance, but I just wanted to point out that sometimes going point-free hurts readability, so that should be taken into account when refactoring.